03-Login-With-Profile/README.md

```diff
@@ -0,0 +1,108 @@
+# Using Passage with Python
+
+Passage provides a Python package to easily authenticate HTTP requests. This folder is an example of how to use Passage in a Python Flask web application. It uses the Passage Auth element for login and registration and the Passage Profile element for the user dashboard.
+
+## Configure Environment Variables
+
+1. Get your Passage App ID and an API Key from the [Passage Console](https://console.passage.id).
+2. Fill in the placeholder variables in `.flaskenv` with these values.
+
+## Configuring a Flask Server
+
+The [passage-identity](https://pypi.org/project/passage-identity/) package depends on a PASSAGE_APP_ID environment variable being set. An application's ID can be copied off of the Passage Console. In this example app, we use the environmnet variable to initialize the Passage SDK and in the HTML template for the login page.
+
+To run this example app, make a virtual environment and run the following commands (the environment variables are stored in `.flaskenv`).
+
+```bash
+Go ahead and create a virtual environment by typing
+>> python3 -m venv venv
+
+Once it is created, you activate the environment by using:
+>> source venv/bin/activate
+
+Install Python dependencies
+>> pip install -r requirements.txt
+
+Run the web server
+>> flask run
+```
+
+## Run With Docker
+
+Make sure you have [docker installed on your computer](https://docs.docker.com/get-docker/).
+
+Create your docker image:
+
+```bash
+$ docker build -t example-python .
+```
+
+Run your docker container using the example-python image, changing the port numbers as necessary:
+
+```bash
+$ docker run -p 5000:5000 example-python
```

Comment:
Note that if you decide to change the container name to follow the `-1`, `-2`, etc. convention, you'll also need to change this